mapie/tests/test_regression.py

```diff
@@ -590,6 +592,24 @@ def test_conformity_score(
     mapie_reg.predict(X, alpha=0.05)
 
 
+@pytest.mark.parametrize(
+    "conformity_score", [ConformalResidualFittingScore()]
+)
+def test_conformity_score_with_split_strategies(
+   conformity_score: ConformityScore
+) -> None:
+    """
+    Test that any conformity score function that handle only split strategies
+    with MAPIE raises no error.
+    """
+    mapie_reg = MapieRegressor(
+        conformity_score=conformity_score,
+        **STRATEGIES["split"]
+    )
+    mapie_reg.fit(X, y + 1e3)
```

Comment:
QUESTION: why do you add 1e3 to y?

Reply:
It was already like that in the test file